csrankings-i.csv

```diff
@@ -210,6 +210,7 @@ Ingrid Wetzel,University of Hamburg,https://www.inf.uni-hamburg.de/de/inst/ab/it
 Ingrid Zukerman,Monash University,http://users.monash.edu/~ingrid,DFyOHW4AAAAJ
 Inhwan Lee,Hanyang University,http://csl.hanyang.ac.kr,NOSCHOLARPAGE
 Inkyoung Hur,Nova Southeastern University,https://cec.nova.edu/faculty/Hur-Inkyoung.html,NOSCHOLARPAGE
+Inna Partin-Vaisband,University of Illinois at Chicago,https://ece.uic.edu/profiles/inna-partin-vaisband-phd/,visXuWkAAAAJ

```

Comment:
ECE, not CS, not clear how they meet inclusion criteria.

Reply:
Dr. Partin-Vaisband is adjunct faculty in CS - https://cs.uic.edu/faculty-staff/adjunct-faculty/.